src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -122,6 +123,13 @@ export class Feedback implements BaseEntity {
         return that.text.startsWith(FEEDBACK_SUGGESTION_IDENTIFIER);
     }
 
```

Comment:
Any reason to name the parameter „that“ rather than „feedback“?

Reply:
I decided to maintain the naming style used in other similar methods, although I'm not sure why it was originally like that.